css/react-datetime.css

```diff
@@ -16,6 +16,10 @@
   box-shadow: 0 1px 3px rgba(0,0,0,.1);
   border: 1px solid #f9f9f9;
 }
+.rdtUp{
+  position: absolute;
+  top: -306px;
+}
```

Comment:
Adding  these 2 styles to `rdtPicker` would fix the open in top problem reliably. Meaning that the popup will always be in the correct place no matter if in calendar or time view which have different heights. The static -306px top position would only work for one of the views.

```
.rdtPicker {
  width: 100%;
  bottom: 100%;
}
```

Reply:
ok maybe this can solve the problem of going out of the screen, if anyone want the datepicker above the input field, this prop can be helpful, a little refraction may be needed.